### ISSUE TITLE
Paperwallet does not have private key?

### ISSUE THREAD

**Issue by mrees999** [ryepdx/ethaddress.org#20]
Trying out the paper wallet but the private key is blurred out cannot recover the eth I sent to it as a test.  Further attempts to generate also show only blurred out private key.  

How do I sweep it back?

Here's the public side.

![public key ether](https://cloud.githubusercontent.com/assets/4064476/13481535/b3c657f4-e0a3-11e5-89b6-d0d02ef9a8ce.JPG)

**Comment by ryepdx** [comment]
The private key is blurred out until you move your mouse over it in the browser. It should de-blur when you print it too. There was an issue with an older version where in Chrome it wouldn't de-blur when it was printed, but I fixed that somewhat recently. What browser are you using? And are you using the website offline? If so, are you using the latest version?

**Comment by mrees999** [comment]
Hi sorry I didn't respond to this earlier - this is one of my less seldomly used email accounts.
What I was trying to do was actually take a screen shot of the private wallet key portion and post it as kind of a 'treasure' hunt with a few dollars worth of eth.  I've done this on occation for bitcoin wallets to get people thinking about it and the possiblities. It's a great conversation starter.  I didn't actually try printing out the wallet.
My memory is a little foggy, but was I supposed to bookmark the URL that it creates for me, or if I've turned off my pc or whatever, is there anyway to find out what that address is again and try to print it out?  One eth won't meen the end of the world for me but it would be nice to finish my goal if it is still possible. If not - I guess I donated it for the good of all ethereum users.  :-)
Thanks!


Date: Wed, 2 Mar 2016 23:59:33 -0800
